packages/bots/fuse-liquidator-bot/src/liquidateUnhealthyBorrows.ts

```diff
@@ -8,6 +8,19 @@ export default async function liquidateUnhealthyBorrows(midasSdk: MidasSdk, retr
   if (retries >= 10) {
     throw "10 retries fetching liquidations, exiting";
   }
+
+  // console.log(`fns ${JSON.stringify(midasSdk.contracts.FuseSafeLiquidator.interface.functions)}`);
+  //
+  // const method = "justTesting";
+  // const params: FuseSafeLiquidator.InputVarsStruct =
+  //   {
+  //     justAddress: "0xF436D47F962e4AcD96Eb6D87677db91D5f40A204",
+  //   }
+  // ;
+  // const data = midasSdk.contracts.FuseSafeLiquidator.interface.encodeFunctionData(method, [params]);
+  //
```

Comment:
@carlomazzaferro  this is actually the reason I was hesitant if the liquidation strategies are ready to be tested - the call to `encodeFunctionData` fails at runtime with any params that I give to it. Probably will not be resolved this evening

Reply:
Hmmmm, I see. Encoding the struct is causing the problem right? Not sure how it is usually done

Reply:
This turned out to be hardly resolvable in a local mainnet forking testing environment. It should be a lot easier to first deploy it and then test it with a hh local mainnet fork.